### ISSUE TITLE
autofocus and PlayBack of taken Pictures

### ISSUE THREAD

**Issue by freischwimmer-bw** [obs1dium/BetterManual#12]
Hi 
I´m using the alpha 5100 for familiy shootings. Ohne thing I really miss is the minimum shutter speed in aperture mode. You gave it to me with this App. Thank you very much!
Your settings works even in normal aperture priority mode  after leaving the app. So i can use autofocus. Unfortunately the Setting ist dismissed after Restart or StandBy. 
May you add autofocus / the other focus modes to the your APP? 
Is there a way to shortly review the shooten images (not automatically, only after pressing the play button)?.

I do programming in VBA and have some basic knowledge of java. If i want to contribute to your project - whats the best was to start with? Generell Android Programming?

**Comment by obs1dium** [comment]
Yes, unfortunately the setting gets lost when the camera shuts down.

The app wasn't meant to fully replace the standard camera UI, hence the name.

As for development, you'll need at least basic Android knowledge. Have a look at #3 (slightly outdated, debugging via ADB should be possible now, but I haven't tried that yet).
Maybe use [KillerInk's fork](https://github.com/KillerInk/BetterManual) as a starting point, who spent a lot of time cleaning up and refactoring the code.
